readme.md

```diff
@@ -2,6 +2,8 @@
 
 This is a repo dedicated to the iOS codebar Workshop. These materials can be used during a normal codebar workshop as long as a coach is there with the skill set.
 
+These tutorials are hosted at https://codebar.github.io/ios-tutorials/
```

Comment:
Could we make this an actual link please so change the markdown to [https://codebar.github.io/ios-tutorials/](https://codebar.github.io/ios-tutorials/) ?

Reply:
🤦‍♂ 

Of course :)

Reply:
Thank you!